_posts/software/2020-10-18-autograd.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "autograd"
+date: 2020-10-18
+author: "@tabakg"
+annotate_criteria: https://rseng.github.io/software/repository/github/HIPS/autograd/annotate-criteria/
+annotate_taxonomy: https://rseng.github.io/software/repository/github/HIPS/autograd/annotate-taxonomy/
+categories:
+- Software
+---
+
+Have you ever wanted to use automatic differentiation but your code is already written in numpy? Or just don't want to learn another library? Today we are showcasing
+ <a href="https://github.com/HIPS/autograd" target="_blank">HIPS/autograd</a>, which will allow you to do just that. That's right, just write a function in numpy, and get all of its gradients for free.
+
+
+<br>
+
+![{{ site.baseurl }}/assets/img/posts/showcase/tanh.png]({{ site.baseurl }}/assets/img/posts/showcase/tanh.png)
+
+<br>
+
+We encourage you to contribute to the [research software encyclopedia](https://rseng.github.io/rse/tutorials/annotation/) and annotate the respository:
+
+<ul>
+<li><a href="{{ page.annotate_criteria }}" target="_blank">Annotate the software criteria</a></li>
+<li><a href="{{ page.annotate_taxonomy }}" target="_blank">Annotate the software taxonomy</a></li>
+</ul>
+
+otherwise, keep reading!
+
+
+<a id="what-is">
```

Comment:
Can you please add back in the small navigation that pointed to these sections? E.g., "What is Autograd," etc

Reply:
Added back